tap_workday/schemas/financial_management_journal_sources.json

```diff
@@ -2,95 +2,89 @@
   "type": "object",
   "properties": {
     "Journal_Source_Reference": {
-      "type": [
-        "null",
-        "object"
-      ],
+      "type": "object",
       "properties": {
         "ID": {
-          "type": [
-            "null",
-            "array"
-          ],
+          "type": "array",
           "items": {
             "type": "object",
             "properties": {
-              "type": {
+              "_value_1": {
                 "type": [
-                  "null",
-                  "string"
+                  "string",
+                  "null"
                 ]
               },
-              "value": {
+              "type": {
                 "type": [
-                  "null",
-                  "string"
+                  "string",
+                  "null"
                 ]
               }
             }
           }
         },
         "Descriptor": {
           "type": [
-            "null",
-            "string"
+            "string",
+            "null"
           ]
         }
       }
     },
     "Journal_Source_Data": {
-      "type": [
-        "null",
-        "object"
-      ],
-      "properties": {
-        "Journal_Source_ID": {
-          "type": [
-            "null",
-            "string"
-          ]
-        },
-        "Journal_Source_Name": {
-          "type": [
-            "null",
-            "string"
-          ]
-        },
-        "Source_For_Accounting_Journal": {
-          "type": [
-            "null",
-            "boolean"
-          ]
-        },
-        "Process_Award_Costs": {
-          "type": [
-            "null",
-            "boolean"
-          ]
-        },
-        "Source_For_Ad_Hoc_Bank_Transaction": {
-          "type": [
-            "null",
-            "boolean"
-          ]
-        },
-        "Enable_Suspense_Processing_for_Web_Service": {
-          "type": [
-            "null",
-            "boolean"
-          ]
-        },
-        "Suspense_Threshold_Percent": {
-          "type": [
-            "null",
-            "number"
-          ]
-        },
-        "Source_For_Workday_Operational_Journal": {
-          "type": [
-            "null",
-            "boolean"
-          ]
+      "type": "array",
```

Comment:
similar question for all array if we can have a "null", "array" which handles for empty array.